cotton/ansible/ansibleutils.py

```diff
@@ -36,7 +38,7 @@ def run_ansible_playbook(
             local('ansible-playbook -i hosts site.yml -e "' + extra_vars + '"')
             # run provisioning playbook
             playbook_cmd = 'ansible-playbook -i hosts '
-            cmd_suffix = ' -e "{}" -v'.format(extra_vars)
+            cmd_suffix = ' -e "{}" -{}'.format(extra_vars, debug)
```

Comment:
my concern here is what if someone sets debug to `None` or `''` how will ansible handle `-e <args> -`